protoc-gen-swagger/options/openapiv2.proto

```diff
@@ -132,8 +147,13 @@ message JSONSchema {
   reserved 1;
   // field 2 is reserved for '$schema', omitted from OpenAPI v2.
   reserved 2;
-  // field 3 is reserved for '$ref', although it is unclear how it would be used.
-  reserved 3;
+  // Ref is used to define an external reference to include in the message.
+  // This should be a fully qualified message reference, and that type must be imported
```

Comment:
> This should be

This *could* be, I suppose?

Reply:
I've adjusted it accordingly. I hope together with the example it is obvious enough.